docs/_ug/commandSummary/itemCommands/find.md

```diff
@@ -1,2 +1,3 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>find n/ITEM_NAME</b><br><br><u>Valid Example:</u><br>find n/Potato
+find ITEM_NAME [ITEM_NAME]
```

Comment:
Just to confirm, is it `n/ITEM_NAME` or just simply `ITEM_NAME` without any prefix?

Reply:
Our find command does not require the n/ prefix/flag. It just searches for matching keywords in the items' names